### DISCUSSION TITLE
How does auto train work?

### DISCUSSION THREAD

**Discussion by KauzDs** [Sau1707/ModernBot#35] [Q&A]
How does auto train work? Does it take X number I input and recruits in X batches or does is it recruit until there is X amount of troops in the town?

**Comment by Sau1707** [comment]
The number that is set it's the final number in the town, if the missing troops are more than a full storage, wait for 80% of the storage before placing them in the quee, else does that as soon as there are enough resources.

It kind of work with the bonus and everything, but sometimes bight be buggy (with some heroes for example)

If the spell option is active, only troop on towns that have the spell active. You can see the active towns in the town list, it will show up an icon next to the town
